assignments/closure.js

```diff
@@ -1,6 +1,27 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+let nigeriaPopulation = "196M";
+
+function nigeria() {
+  const population = 190000000;
```

Comment:
Here we could have used our global `nigeriaPopulation` variable to show that the `nigeria` function has access to the global scope.